Feature: Add a magic submodule to the MDSplus python module

tfredian opened the pull request:
This enhancement enables the use of %tcl and %tdi magic lines
in an iPython or jupyter notebook. You will be able to intersperse
mdstcl commands and tdi expressions in an ipython notebook along
with python.

tfredian commented:
oops. forgot packaging changes. Will retry tomorrow.

joshStillerman commented:
Tom -
  Check that you can still import MDSplus from a python that is not IPython  and on a system that does not have IPython installed.

tfredian committed:
Feature: Add a magic submodule to the MDSplus python module

This enhancement enables the use of %tcl and %tdi magic lines
in an iPython or jupyter notebook. You will be able to intersperse
mdstcl commands and tdi expressions in an ipython notebook along
with python.
